.github/workflows/deploy.yml

```diff
@@ -0,0 +1,28 @@
+name: deploy
+
+on:
+  push:
+    tags:
+      - "v*"
```

Comment:
Can the pattern be a little more specific like a regex `^v([0-9]+\.){2}[0-9]+-pypi` so it's obvious the intent and one is much less likely to release to pypi by mistake.

Reply:
In pytest we use `"[0-9]+.[0-9]+.[0-9]+"`, as it is more usual to use the version directly as tags -- having a `pypi` suffix to the tag is a bit unusual.

I suggest we use `"v[0-9]+.[0-9]+.[0-9]+"` here, which matches the current tags in the repository, and is unlikely to be a mistake. What do you think?

Reply:
That's fine